test/reputation-update.js

```diff
@@ -204,8 +167,7 @@ contract("Colony Reputation Updates", () => {
       });
       await metaColony.finalizeTask(taskId1);
       let repLogEntryWorker = await inactiveReputationMiningCycle.getReputationUpdateLogEntry(3);
-      const result = web3Utils.toBN("1").mul(WORKER_PAYOUT);
-      assert.equal(repLogEntryWorker[1].toString(), result.toString());
+      assert.equal(repLogEntryWorker[1].toNumber(), 300 * 1e18);
```

Comment:
Casting these to a number means it only works because we have a lot of 0s on the end. We would have a loss of precision if that were not true. I'm not going to ask you to change these back, but worth remembering that there is a reason we need to use BigNumber. The cast to string was so that assert had to primitives to compare (two different BigNumber objects are not the same, even if they represent the same value), and they are guaranteed to have whatever precision we need (i.e. infinite).

Reply:
That's fair, I can make this a Big Number.